esrally/mechanic/provisioner.py

```diff
@@ -295,8 +295,8 @@ def variables(self):
             # this is the IP address that the node will be bound to. Rally will bind to the node's IP address (but not to 0.0.0.0). The
             # reason is that we use the node's IP address as subject alternative name in x-pack.
             "network_host": network_host,
-            "http_port": "%d-%d" % (self.http_port, self.http_port + 100),
-            "transport_port": "%d-%d" % (self.http_port + 100, self.http_port + 200),
+            "http_port": str(self.http_port),
```

Comment:
Shouldn't we do the same also in `DockerProvisioner`?

Reply:
I guess this makes sense for consistency; I skipped it because inside a container the starting port will be always available (and we publish only one port externally).

Reply:
Fixed in 02c77bea34e7dbf47e5a0df6eaff5092d80bf60c